### PR TITLE
add new test cases

### PR DESCRIPTION
**Pull request by anqa14**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
         assignToSwimmingCourse: function (name, birthDate) {
             var course;
             var age = this.calculateAge(birthDate);
-
```
Comment (anqa14) [inline app/app.js]: remove consol.log

File: app/app.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+            console.log(name);
```
Comment (anqa14) [inline app/app.js]: remove console.log

Comment (aniaw) [inline app/app.js]: remove console.log

File: app/app.js [hunk 3]
```diff
@@ -21,11 +21,12 @@
             return course;
         },
         calculateAge: function (dateString) {
+            console.log(dateString);
```
Comment (aniaw) [inline app/app.js]: remove console.log

File: app/app.js [hunk 4]
```diff
@@ -21,11 +21,12 @@
...
             var today = new Date();
             var birthDate = new Date(dateString);
             var age = today.getFullYear() - birthDate.getFullYear();
             var m = today.getMonth() - birthDate.getMonth();
-            if (m < 0 || (m === 0 && today.getDate() < birthDate.getDate())) {
+            if (m < 0 || (m == 0 && today.getDate() < birthDate.getDate())) {
```
Comment (aniaw) [inline app/app.js]: why did you change === to ==?
